lib/experimental/fonts/font-library/font-library.php

```diff
@@ -141,8 +128,11 @@ function wp_unregister_font_collection( $collection_id ) {
 	}
 }
 
-// TODO: update to production font collection URL.
-wp_register_font_collection_from_json( 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/01aa57731575bd13f9db8d86ab80a2d74e28a1ac/releases/gutenberg-17.6/collections/google-fonts-with-preview.json' );
+function gutenberg_register_font_collections() {
+	// TODO: update to production font collection URL.
+	wp_register_font_collection( 'google-fonts', 'https://raw.githubusercontent.com/WordPress/google-fonts-to-wordpress-collection/01aa57731575bd13f9db8d86ab80a2d74e28a1ac/releases/gutenberg-17.6/collections/google-fonts-with-preview.json' );
```

Comment:
What's the story of this random URL? Can we replace it with a build script or something? I'm sure that including this into WordPress Core is a no go.

Reply:
This URL is provisional until the last version of [the file is deployed WordPress.org CDN](https://s.w.org/images/fonts/17.6/collections/google-fonts-with-preview.json).

Reply:
> random URL?

Do you refer to the domain of the URL or the fact that is loading from an external URL?

Reply:
If it's WordPress.org CDN, I'm ok with that, I believe we already use that CDN in a few places on WordPress already. 

Should we just use the outdated version for now? I mean should we make the change in Gutenberg (and the Core patch)

Reply:
Some fields were added to the data of the file. Until the current file version is deployed to the CDN, we must use a file with all the required fields. As soon as we get the new file deployed to the CDN, we can replace that URL.

Reply:
@matiasbenedetto What's blocking the new file from being uploaded to the CDN?

Reply:
The latest changes will be merged soon and after that we will request to deploy the changes in this track ticket: https://meta.trac.wordpress.org/ticket/7282#ticket

Reply:
The deploy to the CDN was already requested here: https://meta.trac.wordpress.org/ticket/7282#comment:17

Reply:
For now, I've updated the url to the release version of the collection in Github.

Reply:
Reverted, I did that too soon, as the preview files aren't loaded at s.w.org yet.